api/src/node/constants.ts

```diff
@@ -21,12 +30,11 @@ export {
     TEAM_SPECTATOR,
     TEAM_CT,
     TEAM_T,
-    HUD_PRINT_TALK,
-    HUD_PRINT_CONSOLE,
-    HUD_PRINT_CENTER,
     STEAM_USER_HIGH_VALUE,
     GAME_MESSAGE_TARGET_CONSOLE,
-    GAME_MESSAGE_TARGET_HUD
+    GAME_MESSAGE_TARGET_HUD,
+    GAME_MESSAGE_TARGET,
+    PLAYER_TEAM
 }
 
 export default {
```

Comment:
You should add export to the individual things instead of exporting all of them as a default object. Not really how this was intended to be used.